resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ResourceValidator.scala

```diff
@@ -97,7 +97,7 @@ object ResourceValidator {
         workerProcess
       ).covary[F]
 
-      _ <- processes.parJoinUnbounded
+      _ <- processes.parJoin(9)
```

Comment:
Seems there are 11 sub-streams.. should this be parJoin(11)?

Reply:
yea, I intentionally made this 9, hoping reducing the number of Streams running in parallel will make the connection error less often